guides/v2.2/cloud/project/project-conf-files_services.md

```diff
@@ -6,93 +6,78 @@ functional_areas:
 
 elasticsearch:
-    type: elasticsearch:6.5
+    type: elasticsearch:<version>
     disk: 1024
 ```
 
-Modify this file to use specific and additional services in your deployment. See the [`type`](#cloud-yaml-services-type) section to see the services we support and deploy for you if you add them to the file.
-
-## Service values {#services}
+## Service values
 
-To add a service, you add the following data to services.yaml:
+You must provide name and type for each service, and a disk value if the service uses persistent storage. For example:
 
 ```yaml
 name:
-    type: name:version
+    type: name:<version>
     disk: value
 ```
 
-For example:
-
-```yaml
-mysql:
-    type: mysql:10.0
-    disk: 2048
-```
-
-### `name` {#cloud-yaml-services-name}
-
-Identifies the service in the project. The `name` can consist only of lower case alphanumeric characters: `a` to `z` and `0` to `9`. For example, Redis is entered as `redis`.
+### `name`
 
-You can have multiple instances of each service type. For example, you could have multiple Redis instances. For example, we use multiple Redis instances, one for session and one for cache.
+The `name` value identifies the service in the project and consists only of lower case alphanumeric characters: `a` to `z` and `0` to `9`, such as `redis`. You can have multiple instances of each service type. For example, we use multiple Redis instances, one for session and one for cache.
 
 ```yaml
 redis:
-    type: redis:3.0
+    type: redis:<version>
 
 redis2:
-    type: redis:3.0
+    type: redis:<version>
 ```
 
-Be aware, if you rename a service in the `services.yaml` file, the following is **permanently removed**:
+Renaming a service in the `services.yaml` file **permanently removes** the following:
 
 -  The existing service before creating a new service with the new name you specify.
 -  All existing data for the service is removed. We strongly recommend you [snapshot your environment]({{ page.baseurl }}/cloud/project/project-webint-snap.html) before you change the name of an existing service.
 
-### `type` {#cloud-yaml-services-type}
-
-The `type` of your service in the format: `type:version`
+### `type`
 
-We support and deploy the following services for you:
-
--  [`mysql`]({{ page.baseurl }}/cloud/project/project-conf-files_services-mysql.html)
--  [`redis`]({{ page.baseurl }}/cloud/project/project-conf-files_services-redis.html)
--  [`elasticsearch`]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html)
--  [`rabbitmq`]({{ page.baseurl }}/cloud/project/project-conf-files_services-rabbit.html)
+The `type` value specifies the version of your service. Use the format: `type:version`
 
-### `disk` {#cloud-yaml-services-disk}
+### `disk`
 
-Specifies the size of the persistent disk storage (in MB) allocated to the service.
+The `disk` value specifies the size of the persistent disk storage (in MB) to allocate to the service. Services that use persistent storage must provide a disk value.
```

Comment:
actually, there is only one service now which is stored in memory - Redis. and maybe describe it like
```
The `disk` value specifies the size of the persistent disk storage (in MB) to allocate to the service. Every service which uses persistent storage must provide a disk value, when for Redis where all data is in memory, you should not set `disk` field. 
```